driver-core/src/main/java/com/datastax/driver/core/EventDebouncer.java

```diff
@@ -117,6 +117,12 @@ void stop() {
         break;
       }
     }
+    while (true) {
```

Comment:
Isn't there a risk of infinite loop?

Reply:
No, this is the typical idiom for CAS operations; the `break` instruction inside the loop guarantees that the thread will eventually exit the loop, even if it takes a few times to get the CAS operation to succeed.